fastlane/swift/SocketResponse.swift

```diff
@@ -28,6 +28,7 @@ struct SocketResponse {
             }
             
             if status == "ready_for_next" {
+                print("ready for next")
```

Comment:
Either delete or 👇 ?

```suggestion
                verbose(message: "ready for next")
```